website/content/fr/wgs/artifacts/charter/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Artifacts-WG Charter
+url: 'wgs/artifacts/charter/charter.md/'
```

Comment:
Just learned that this URL line causes issues in the Spanish translation. 

Could you look to remove it? I believe there are a few instances of it across the docs. 

See: https://github.com/cncf/tag-app-delivery/commit/428df6d803c773e993eb88c63fb9398871a0f07f

Reply:
Thanks for catching and flagging this. Now taken into account.